Dockerfile

```
@@ -1,7 +1,7 @@
 # This file is auto generated from it's template,
 # see citusdata/tools/packaging_automation/templates/docker/latest/latest.tmpl.dockerfile.
-FROM postgres:13.3
-ARG VERSION=10.0.4
+FROM postgres:13.2
```

```suggestion
FROM postgres:13.3
```

Let's keep this as is, no need to downgrade

---

Fixed